src/python/WMCore/MicroService/DataStructs/MSRuleCleanerWflow.py

```diff
@@ -70,6 +70,8 @@ def docSchema(self):
             ('ParentageResolved', True, bool),
             ('PlineMarkers', None, list),
             ('IsClean', False, bool),
+            ('IsLogDBClean', False, bool),
+            ('IsArchDelayExpired', False, bool),
```

Comment:
I would completely write this name, such as `IsArchivalDelayExpired`

Reply:
Done